app/services/tunnel.py

```diff
@@ -96,26 +100,35 @@ def over_tunnel_limit(self) -> bool:
 
     def create_tunnel_nomad(self) -> Tuple[str, dict]:
         """Create a tunnel by scheduling an SSH container into the Nomad cluster"""
+        tcp_ports = []
+        for port in self.port_types:
+            if "tcp" in port:
+                tcp_ports.append(self.get_tcp_port())
+            else:
+                tcp_ports.append(0)
+        stripped_ssh_key = "".join(i for i in self.ssh_key if 31 < ord(i) < 127)
```

Comment:
It's not obvious what this does - I think it strips high-range bytes out of an SSH public key string?  Why do we need to do that?